demo/runners/agent_container.py

```diff
@@ -1109,6 +1136,9 @@ async def test_main(
             if consortiq_container:
                 log_msg("Shutting down consortiq agent ...")
                 await consortiq_container.terminate()
+            if airops_container:
+                log_msg("Shutting down consortiq agent ...")
+                await consortiq_container.terminate()
```

Comment:
would this be 'shutting down airops' and airops_container.terminate()?

does `caa_container` also need terminating?